mix/mix_protocol.nim

```diff
@@ -413,8 +528,17 @@ proc anonymizeLocalProtocolSend*(
     (destPeerId.isNone and fwdDestination.isSome),
     "specify either the destPeerId or destination but not both"
 
+  var config = SendPacketConfig(logType: Entry)
+  when defined(enable_mix_benchmarks):
+    config.startTime = Moment.now()
+
   let (multiAddr, _, _, _, _) = getMixNodeInfo(mixProto.mixNodeInfo)
 
+  when defined(enable_mix_benchmarks):
+    config.orig = uint64.fromBytesLE(msg[5 ..< 13])
+    config.msgid = uint64.fromBytesLE(msg[13 ..< 21])
+    config.origAndMsgId = msg[5 ..< 21]
```

Comment:
Minor suggestion: IIRC, this assumes a fixed GossipSub message layout (`message size = 100`) as defined in [tmp/mix-gossipsub-logging branch](https://github.com/vacp2p/nim-libp2p/tree/tmp/mix-gossipsub-logging). Might be worth adding a short comment or defining constants just to make that assumption clearer.